example/ios/Runner/Info.plist

```diff
@@ -36,6 +36,12 @@
 	</array>
 	<key>NSLocalNetworkUsageDescription</key>
 	<string>Uses WiFi to connect and sync with nearby devices</string>
+	<key>LSSupportsOpeningDocumentsInPlace</key>
```

Comment:
Does this need corresponding entitlements in both macos entitlements files?

Reply:
It's working with the existing entitlements in MacOS when I tested.